packages/odata-service-inquirer/src/prompts/datasources/sap-system/service-selection/service-helper.ts

```diff
@@ -284,13 +284,15 @@ export function getSelectedServiceLabel(username: string | undefined): string {
  * @param selectedService the selected service
  * @param connectValidator the connection validator
  * @param requiredOdataVersion the required OData version for the service
+ * @param hasAnnotations
```

Comment:
Please add a param description explaining the use of the parameter

Reply:
Thank you @IainSAP , I have added a comment